test/models/content_view_version_test.rb

```diff
@@ -67,6 +68,32 @@ def test_docker_count
       assert_equal image_count, cvv.docker_image_count
       assert_equal tag_count, cvv.docker_tag_count
     end
+
+    def test_search_equal_version
+      assert_includes ContentViewVersion.search_for("version = 1.0"), @cvv
+      assert_includes ContentViewVersion.search_for("version = 1"), @cvv
```

Comment:
What about an assert that ensures a search of just the major returns all the minors as well?